### PR TITLE
New `add handler`

### PR DESCRIPTION
**Pull request by chgeo**
In get-started:
- Hint users to new `cds add handler` to create the `js` file automatically.
- Adjust the `js` sample code to 'new' class-style. This is also what the linked sample code uses.

In CLI docs:
- improve help section for `add handler`
- add full help text for `cds add --help`.  This gets synchronized.

### CODE REVIEW

File: tools/assets/help/cds-add.out.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+      --plan
+
+        Specify the service plan.
+
+
+    <em>cds add cloud-logging</em>
+
+      --plan
+
+        Override the service plan used for the MTA generation.      --with-telemetry
```
Comment (renejeglinsky) [inline tools/assets/help/cds-add.out.md:83]: That seems misplaced somehow

Comment (chgeo) [reply tools/assets/help/cds-add.out.md:83]: Good catch

Comment (chgeo) [reply tools/assets/help/cds-add.out.md]: Fixed

**Review by renejeglinsky — APPROVED** [review]

File: tools/cds-cli.md [hunk 2]
```diff
@@ -319,31 +323,37 @@ assumes a remote app named `bookshop` on CloudFoundry and a JWT token for this a
...
-```sh
+To generate handler files, run:
+
+::: code-group
+```sh [Node.js]
+cds add handler
+```
+```sh [Java]
+mvn compile  # let Java know what your model looks like
+cd srv       # folowing must be run from srv/
```
Comment (swaldmann) [inline tools/cds-cli.md:338]:
```suggestion
cd srv       # following must be run from srv/
```

File: get-started/in-a-nutshell.md [hunk 3]
```diff
@@ -600,6 +600,10 @@ bookshop/
 [Learn more about providing service implementations **in Node.js**.](../node.js/core-services#implementing-services){.learn-more .impl .node}
 [Learn also **how to do that in Java** using Event Handler Classes.](../java/event-handlers/#handlerclasses){.learn-more .impl .java}
 
+::: tip
+You can have this _.js_ file created automatically with [`cds add handler`](../tools/cds-cli#handler).
+:::
```
Comment (swaldmann) [inline get-started/in-a-nutshell.md:605]: Even if low information-density here, I like a heading more than the default uppercase "TIP":
```suggestion
::: tip Use `cds add handler`
You can have this _.js_ file created automatically with [`cds add handler`](../tools/cds-cli#handler).
:::
```

Comment (swaldmann) [reply get-started/in-a-nutshell.md:605]: Or (as in old CAPire iirc) we hide the title if not specified

**Review by swaldmann — APPROVED** [review]
Only minor comments

**Pull request merged**